docs/destinations/forwarding-events/snowbridge/configuration/transformations/testing/index.md

```diff
@@ -0,0 +1,85 @@
+`docker run snowplow/snowbridge:${versions.snowbridge}`
+}</CodeBlock>
+
+Since the default configuration for snowbridge is already `stdin` source and `stdout` target. So, to test sending data through with no transformations, (where `data.tsv` contains our test data) we can simply run:
+
+<CodeBlock language="bash">{
+`cat data.tsv | docker run -i snowplow/snowbridge:${versions.snowbridge}`
+}</CodeBlock>
+
+This will print the data to console, along with logs - note that the metrics reported in logs may report that no data has been processed. This is because the app reached the end of output and exited before the default reporting period of 1 second.
+
+It is likely easiest to output the results to file, to make it easier to examine the results:
+
+<CodeBlock language="bash">{
+`cat data.tsv | docker run -i snowplow/snowbridge:${versions.snowbridge} > output.txt`
+}</CodeBlock>
+
+Note that the resulting data will contain more than the data: `PartitionKey`, `TimeCreated` `TimePulled` and `TimeTransformed` will also be present. The data that reaches the target in a production setup is under `Data`.
+
+### Adding configuration
+
+Adding the specific configuration to test is a matter of mounting a configuration file, and setting the `SNOWBRIDGE_CONFIG_FILE` env var to point to that path. If your configuration is in `config.hcl`, you can add configuration to the above example as follows:
+
+<CodeBlock language="bash">{
+`cat data.tsv | docker run -i \\
+    --mount type=bind,source=$(pwd)/config.hcl,target=/tmp/config.hcl \\
+    --env SNOWBRIDGE_CONFIG_FILE=/tmp/config.hcl \\
+    snowplow/snowbridge:${versions.snowbridge} > output.txt`
+}</CodeBlock>
+
+Note that docker expects absolute paths for mounted files - here we use pwd but you can specify the aboslute path manually too.
+
+To test transformations, your configuration file only needs to contain the `transform` block(s) that you're testing, leaving the source and targets on default (by not specifying them in the configuration file).
+
+### Adding custom scripts
+
+Finally, you can add custom scripts by mounting a file, in the exact same manner. Assuming the script is in `script.js`, that looks like this:
+
+<CodeBlock language="bash">{
+`cat data.tsv | docker run -i \\
+    --mount type=bind,source=$(pwd)/config.hcl,target=/tmp/config.hcl \\
+    --mount type=bind,source=$(pwd)/script.js,target=/tmp/script.js \\
+    --env SNOWBRIDGE_CONFIG_FILE=/tmp/config.hcl \\
+    snowplow/snowbridge:${versions.snowbridge} > output.txt`
+}</CodeBlock>
+
+Note that you must ensure that the transformation config points to the mounted path (the `target` in the mount path - `/tmp/script.js` above).
+
+## Further testing
+
+You can use the above method to test sources and targets from a local environment too, which can be useful in combination with transformations - simply set up the appropriate authentication for your source and/or target, add them to your configuration file, and run as directed above.
```

Comment:
```suggestion
You can use the above method to test sources and targets from a local environment too. This can be useful in combination with transformations. Set up the appropriate authentication for your source and/or target, add them to your configuration file, and run the commands from the previous sections.
```